Sources/MapboxCoreNavigation/RouteController.swift

```diff
@@ -252,7 +252,11 @@ open class RouteController: NSObject {
         
         rawLocation = location
         
-        locations.forEach { navigator.updateLocation(for: FixLocation($0)) }
+        locations.forEach {
+            navigator.updateLocation(for: FixLocation($0)) { _ in
```

Comment:
Not sure that we have to do anything here. Additional logging might be helpful, but if there are a lot of locations it might clutter the output. I also think that this callback should be optional.